test/data/example_dot_profile_config.xml

```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 
-<image schemaversion="6.9" name="LimeJeOS-openSUSE-13.2" displayname="schäfer">
+<image schemaversion="6.9" name="LimeJeOS-openSUSE-13.2" displayname="schaefer">
```

Comment:
Why is this changing everywhere?

Reply:
For some obscure reason, the builds with Python 3.4 that I added fail on GitLab, see for instance: https://gitlab.com/D4N/kiwi-ci/-/jobs/171932726.

Python complains about not being able to decode the character `\xe4` (that is "ä" in one of iso-something encodings). Thus I removed the "ä" from @schaefi's name and the tests now pass.

I can of course just remove the Python 3.4 environment and drop this commit.

Reply:
Just drop Python 3.4.